File: 2580-circular-sentence/2580-circular-sentence.ts
function isCircularSentence(sentence: string): boolean {
    let arr = sentence.split(" ")

    for(let i = 0; i < arr.length-1;i++){
      let fs = arr[i]
      let ss = arr[i+1]

      let fl = fs.length

      if(fs[fl-1] !== ss[0]){
        return false
      }
    }

    let lw = arr[arr.length-1]
    let fw = arr[0]

    if(fw[0] !== lw[lw.length-1]){
      return false
    }

    return true
};

File: 2580-circular-sentence/2580-circular-sentence.go
func isCircularSentence(sentence string) bool {
    spl := strings.Split(sentence, " ")

    for i := 0; i < len(spl)-1;i++{
      fs := spl[i]
      ss := spl[i+1]

      if(fs[len(fs)-1] != ss[0]){
        return false
      }
    }

    fw := spl[0]
    lw := spl[len(spl)-1]

    if(fw[0] != lw[len(lw)-1]){
      return false
    }
    return true
}